tokenizer

```diff
@@ -0,0 +1,24 @@
+import numpy as np
+
+def tokenizer(
+        nucleotide_sequence):
+    """
+    tokenizer converts a nucleotide sequence to a sequence of ids.
+
+    nucleotide_sequence -- DNA sequence string consisting of A, C, G, T
+    return -- a x by 2 numpy array consisting of converted nucleotides and their positions
+    """ 
+    arr = [] # Declare an empty array
+    counter = 0 # Declares a counter which gives the position of each nucleotide
+    for x in nucleotide_sequence:
+        if x == "A" or x == "a":
+            arr.append([0,counter]) # Map A to 0
+        elif x == "C" or x == "c":
+            arr.append([1,counter]) # Map C to 1
+        elif x == "G" or x == "g":
+            arr.append([2,counter]) # Map G to 2
+        elif x == "T" or x == "t":
+            arr.append([3,counter]) # Map T to 3
+        counter += 1 # Increment counter
+    np_arr = np.array(arr) # Convert finished array to a numpy array
```

Comment:
Breaks single-responsibility principle. The function name claims it tokenizes a string, not additionally constructing a NumPy array.